src/textField/style.ts

```diff
@@ -1,6 +1,9 @@
 import css from 'styled-jsx/css'
 import { color, space, font, radius, transition } from '_utils/branding'
 
+const borderSize = '1px'
+const borderFocusSize = '3px'
```

Comment:
Shouldn't they be in the `_utils/branding` file?

Reply:
Discussed IRL, IMO, LGTM. We use the 1px border to keep the consistent padding to 16px and have the textfield height to 56px, since it's only used in the textfield we don't want to put it in a global branding file.